sources/models/shared-defaults/containerd-pki.toml

```diff
@@ -0,0 +1,2 @@
+[metadata.settings.pki]
+affected-services = ["pki", "containerd"]
```

Comment:
We need a migration for this (`ReplaceMetadataListsMigration`) since we're changing the list of affected services. However, the problem is that the new value depends on the variant, and migrations previously haven't been variant-specific.

You might be able to make this work with conditional compilation based on the variant, since the migration binaries are part of the same artifacts as the update images - meaning the right migration binary would end up in the right repo.

It'd need to be thoroughly tested though, since it's new territory.

Reply:
> migrations previously haven't been variant-specific.

For the sake of bookkeeping, here is the issue for that #1105